src/test/java/com/wikia/webdriver/common/templates/NewTestTemplateCore.java

```diff
@@ -62,16 +67,11 @@ protected void prepareURLs() {
     wikiCorpSetupURL = urlBuilder.getUrlForWiki("corp");
   }
 
-  protected void startBrowser() {
-    driver = registerDriverListener(
-        NewDriverProvider.getDriverInstanceForBrowser(Configuration.getBrowser()));
-  }
-
   protected void setWindowSize() {
     Dimension browserSize = Configuration.getBrowserSize();
     String browser = Configuration.getBrowser();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "browser" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1481)